cMake/UseLibPack3.cmake

```
@@ -0,0 +1,75 @@
+set(ENV{PATH} "${FREECAD_LIBPACK_DIR};$ENV{PATH}")
+set(ENV{CMAKE_PREFIX_PATH} ${FREECAD_LIBPACK_DIR})
+
+# Make sure we are using the static versions of Boost here: the LibPack includes both
+set(Boost_DIR ${FREECAD_LIBPACK_DIR}/lib/cmake CACHE PATH "" FORCE)
+set(Boost_USE_STATIC_LIBS OFF)
+find_package(Boost COMPONENTS filesystem program_options regex system thread date_time REQUIRED)
```

```suggestion
set(Boost_DIR ${FREECAD_LIBPACK_DIR}/lib/cmake CACHE PATH "" FORCE)
set(Boost_USE_STATIC_LIBS OFF)
find_package(Boost COMPONENTS filesystem program_options regex system thread date_time REQUIRED)
```

`find_package` also has a PATHS argument. So you might not need to set `Boost_DIR` explicitly

---

Yes: I've modified to set the paths as part of find_package().